controllers/ingressnodefirewallconfig_controller_test.go

```diff
@@ -26,7 +26,9 @@ var _ = Describe("Ingress nodefirewall config Controller", func() {
 				},
 			}
 			daemonContainers := map[string]string{
+				"mount-bpffs":     "test-daemon:latest",
```

Comment:
this won't work we need to create new map for daemonInitContainers and add the mount-bpffs there 
then new loop to go over `daemonSet.Spec.Template.Spec.InitContainers`
something like this
```golang
for _, c := range daemonSet.Spec.Template.Spec.InitContainers {
    image, ok := daemonInitContainers[c.Name]
    Expect(ok).To(BeTrue(), fmt.Sprintf("init container %s not found in %s", c.Name, daemonInitContainers))
    Expect(c.Image).To(Equal(image))
}
```

Reply:
OK. I didn't look to closely when I added the initContainer name. Thanks.

Reply:
Done in https://github.com/openshift/ingress-node-firewall/compare/c3372714c7382673aaa25572102f07151c8341f3..46f5a27d3c523a4ad2439511ddeaaefcb6dfcb92.